src/hooks/__tests__/useReporter.test.ts

```diff
@@ -18,9 +18,11 @@ describe('useReporter hook', () => {
   });
 
   it('calls useContext with ErrorBoundaryContext and returns', () => {
-    const context = {};
+    const context = {
+      reporter: console,
+    };
     (useContext as jest.Mock).mockReturnValueOnce(context);
-    expect(useReporter()).toBe(context);
+    expect(useReporter()).toBe(console);
     expect(useContext).toBeCalledWith(ErrorBoundaryContext);
```

Comment:
Honestly I don't think this test serves any purpose - other than improving coverage on paper. It quite literally *tests if the implementation contains the line `return useContext(ErrorBoundaryContext)`*, which is a testing antipattern IMO.

I'd just get rid of the test completely.